packages/nats-runner/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "@jwalab/nats-runner",
+  "version": "0.0.1",
+  "description": "",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "devDependencies": {
+    "@tsconfig/node14": "^1.0.1",
+    "@typescript-eslint/eslint-plugin": "^4.4",
+    "@typescript-eslint/parser": "^4.4",
+    "eslint": "^7.32.0",
+    "eslint-config-prettier": "^8.3.0",
+    "openapi-types": "^9.3.0",
+    "prettier": "^2.3.2",
+    "typescript": "^4.4"
+  },
+  "dependencies": {
+    "jwt-decode": "^3.1.2",
+    "@jwalab/logger": "latest",
+    "@jwalab/open-api": "latest",
+    "node-dependency-injection": "^2.7.1"
+  },
+  "peerDependencies": {
+    "nats": "^2.1.0"
```

Comment:
I'm not sure about that ... we can talk about it

Reply:
it's a good idea to not include NATS as part of the dependencies but still tell which version of NATS this package will work with. I think peerDependencies is the best way to let npn know what version of NATS it's expecting. Happy to hear your thoughts

Reply:
I see the benefit of being able to choose which version you want to work with in a specific service. I see the downside that you have to wonder which version to use and remember to add it yourself in your dependencies.
If we go that way we should do it for other dependencies as well